.github/workflows/integration.yml

```
@@ -12,6 +12,10 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - name: Verify branch
+        if: ${{ github.base_ref != 'main' }}
+        run: |
+          echo "This workflow can only run as-is on the main branch. Please remove this conditional check and make sure GWC and GS are checked out on the right branches for this GeoTools series (see the --branch hints below). Current branch is ${GITHUB_BASE_REF}"
```

```suggestion
        uses: actions/github-script@v7
        script: |
          core.setFailed("This workflow can only run as-is on the main branch. Please remove this conditional check and make sure GWC and GS are checked out on the right branches for this GeoTools series (see the --branch hints below). Current branch is ${GITHUB_BASE_REF}")
```

this might work, according to https://stackoverflow.com/a/68777117/5454667

---

Let's give it a try

---

Apparently not:

```
[Invalid workflow file: .github/workflows/integration.yml#L18](https://github.com/geotools/geotools/actions/runs/10301727692/workflow)
The workflow is not valid. .github/workflows/integration.yml (Line: 18, Col: 9): Unexpected value 'script'
```